cf-usb/cf-usb.yaml

```diff
@@ -30,27 +30,31 @@ resources:
   source:
     repository: ((docker-server))((docker-org))/cf-usb-sidecar
     tag: ((docker-tag))
+    tag_as_latest: true
     username: ((docker-username))
     password: ((docker-password))
 - name: docker.buildbase
   type: docker-image
   source:
     repository: ((docker-server))((docker-org))/cf-usb-sidecar-buildbase
     tag: ((docker-tag))
+    tag_as_latest: true
     username: ((docker-username))
     password: ((docker-password))
 - name: docker.mysql
   type: docker-image
   source:
     repository: ((docker-server))((docker-org))/cf-usb-sidecar-dev-mysql
     tag: ((docker-tag))
+    tag_as_latest: true
     username: ((docker-username))
     password: ((docker-password))
 - name: docker.mysql-setup
   type: docker-image
   source:
     repository: ((docker-server))((docker-org))/cf-usb-sidecar-dev-mysql-setup
     tag: ((docker-tag))
+    tag_as_latest: true
```

Comment:
This should go on the output params, not resource definition, I think

Reply:
Done with commit ef4a90b749f65afd3166761227f8b96c066f4122. In testing.

Reply:
Works.